index.html

```diff
@@ -8,8 +8,20 @@
     <link rel="stylesheet" href="css/index.css">
 </head>
 <body>
+    <!-- Header -->
+    <header>
     <h1>Haregebezu Amare</h1>
-    <section class="about">
+    </header>
+    <!-- nav menu -->
+    <nav class="nav">
+        <ul class="menu-nav">
+            <li class="nav-item"><a class="nav-link" href="#about"> About</a></li>
+            <li class="nav-item"><a class="nav-link" href="#exprience"> Experience</a></li>
+            <li class="nav-item"><a class="nav-link" href="#skills"> Skills</a></li>
```

Comment:
```suggestion
            <li class="nav-item"><a class="nav-link" href="#about">About</a></li>
            <li class="nav-item"><a class="nav-link" href="#exprience">Experience</a></li>
            <li class="nav-item"><a class="nav-link" href="#skills">Skills</a></li>
```

Probably don't want extra space inside `<a>`